CMakeLists.txt

```diff
@@ -123,6 +123,7 @@ add_subdirectory(libwebthree)
 add_subdirectory(libweb3jsonrpc)
 
 add_subdirectory(aleth)
+add_subdirectory(aleth-bootnode)
```

Comment:
Maybe this should go under `if (TOOLS)` ?

Reply:
Yes.

Reply:
Thanks for the tip, done!